features/step_definitions/auction_create_steps.rb

```diff
@@ -11,7 +11,7 @@
 end
 
 Given(/^there is a closed auction$/) do
-  @auction = FactoryGirl.create(:auction, :closed, :with_bids)
+  @auction = FactoryGirl.create(:auction, :closed, :with_bids, :c2_budget_approved)
```

Comment:
I thought about that, but I didn't know if I should do that on this PR. Let me try it though